### PR TITLE
Fixed window rendering on resize and white background on start up.

### PR DESCRIPTION
**Pull request by Marco012**
With this PR, the window is able to render and receive events while resizing, instead of leaving a black area. This also opens the window with the background color 'colorBg' instead of a white flicker.

### CODE REVIEW

File: imgui-app/src/main/java/imgui/app/Window.java [hunk 1]
```diff
@@ -90,14 +87,29 @@ protected void initWindow(final Configuration config) {
...
 
         if (config.isFullScreen()) {
             GLFW.glfwMaximizeWindow(handle);
         }
+        else {
+            GLFW.glfwShowWindow(handle);
+        }
 
-        GL.createCapabilities();
+        GL32.glClearColor(colorBg.getRed(), colorBg.getGreen(), colorBg.getBlue(), colorBg.getAlpha());
```
Comment (SpaiR) [inline imgui-app/src/main/java/imgui/app/Window.java:102]: Please, move this logic to a separate method with relatable javadoc about whtat exactly it does and why it's needed.

File: imgui-app/src/main/java/imgui/app/Window.java [hunk 2]
```diff
@@ -90,14 +87,29 @@ protected void initWindow(final Configuration config) {
         }
 
         GLFW.glfwMakeContextCurrent(handle);
+
+        GL.createCapabilities();
```
Comment (SpaiR) [inline imgui-app/src/main/java/imgui/app/Window.java:91]: I'm not sure, why this line was moved here?

**Review by SpaiR — CHANGES_REQUESTED** [review]
Thank you for your contribution! Please, take a look at CI results. Your code violates some of checkstyle rules, which need to be addressed.

File: imgui-app/src/main/java/imgui/app/Window.java [hunk 3]
```diff
@@ -35,7 +37,7 @@
     /**
      * Background color of the window.
      */
-    protected final Color colorBg = new Color(.5f, .5f, .5f, 1);
+    protected Color colorBg = new Color(.5f, .5f, .5f, 1);
```
Comment (SpaiR) [inline imgui-app/src/main/java/imgui/app/Window.java:40]: Why did you remove the `final` modifier`?

Comment (Marco012) [reply imgui-app/src/main/java/imgui/app/Window.java:40]: You weren’t able to modify it since it’s final. How do you change the background color on a class that implements this?

Comment (SpaiR) [reply imgui-app/src/main/java/imgui/app/Window.java:40]: - It's a protected variable, so every implementation can access it.
- There is a public getter for the field, so it could be accessed outside the class.
- `Color` class itself has a bunch of methods to modify its value, like: `Color#set(float, float, float, float)`.

Comment (Marco012) [reply imgui-app/src/main/java/imgui/app/Window.java:40]: Ah ok, I didn’t know about `Color#set(float, float, float, float)`.

**Review by SpaiR — APPROVED** [review]
LGTM! 👍
